src/lib.rs

```diff
@@ -195,17 +274,19 @@ unsafe fn trace_from_process_id(
         .s
         .copy_from_slice(&kernel_logger_name_with_nul[..]);
 
-    let kernel_logger_name_with_nul_pcstr = PCSTR(kernel_logger_name_with_nul.as_ptr());
+    // let kernel_logger_name_with_nul_pcstr = PCSTR(kernel_logger_name_with_nul.as_ptr());
     // Stop an existing session with the kernel logger, if it exists
```

Comment:
I wonder if this can cause UB if blondie is running in multiple processes...

Reply:
Right now it seems to cause
`value: Other(WIN32_ERROR(4201), "The instance name passed was not recognized as valid by a WMI data provider.\r\n", "TraceSetInformation stackwalk")`